test_harness/src/lib.rs

```diff
@@ -28,7 +29,7 @@ pub mod deposits;
 /// NOTE: Current the snapshot and test setup use a single Chain.
 ///
 /// Uses Chain #1337 from the [`GANACHE_CONFIG`] static to init the contracts
-pub static SNAPSHOT_CONTRACTS: Lazy<Contracts> = Lazy::new(|| {
+pub static SNAPSHOT_CONTRACTS_1337: Lazy<Contracts> = Lazy::new(|| {
```

Comment:
We should think of better way to setup and test when using multi-chain. This is sufficient for the time being.